galleries/users_explain/toolkits/axisartist.rst

```diff
@@ -2,9 +2,9 @@
 
 .. _axisartist:
 
-======================
-The axisartist toolkit
-======================
+===  ==========  =======
+The  axisartist  toolkit
+===  ==========  =======
```

Comment:
I assume this is unintended. Have you been playing with table formatting too much? :stuck_out_tongue_winking_eye: